utils/workerpool/workerpool.go

```diff
@@ -0,0 +1,32 @@
+package workerpool
+
+import (
+	"github.com/gammazero/workerpool"
+
+	"github.com/onflow/flow-go/module/component"
+	"github.com/onflow/flow-go/module/irrecoverable"
+)
+
+// WorkerPool is a utility structure which allows wrapping WorkerPool into module.Startable and module.ReadyDoneAware
+// interfaces and allows to use this structure as component.
+type WorkerPool struct {
+	*component.ComponentManager
+	*workerpool.WorkerPool
+}
+
+// New performs initialization of worker pool and setups component manager
+// which will shutdown workers when requested to quit.
+func New(maxWorkers int) *WorkerPool {
```

Comment:
```suggestion
func NewManaged(maxWorkers int) *WorkerPool {
```
Could name this differently from "github.com/gammazero/workerpool" so it's easier to distinguish what kind of workerpool is being created at a glance